posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: "MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in 22.0.0.1-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in 22.0.0.1-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties. 
+blog_description: Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile 5, MicroProfile Rest Client 3.0 and JPA enhancements in Open Liberty 22.0.0.1-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.1-beta offers MicroProfile 5.0 including MicroProfile Rest Client 3.0, which align with Jakarta EE 9.1. Also available in this beta release is an enhancement to JPA which provides the ability to declare "default" JPA persistence properties.
+
+For 22.0.0.1-beta, there is a single beta package for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: this package contains all Open Liberty beta features and GA features and functions.
+
+This means that you can try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you give the beta package a try, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features and enhancements:
+
+* <<mp_5.0, MicroProfile 5.0>>
+* <<rest_client, MicroProfile Rest Client 3.0>>
+* <<jpa_persistence, Define JPA persistence properties at server scope>>
+
+
+[#mp_5.0]
+=== MicroProfile 5.0
+
+MicroProfile 5.0 enables applications to use MicroProfile APIs together with Jakarta EE 9.1. MicroProfile 5.0 does not provide any other functional updates except aligning with Jakarta EE 9.1.  MicroProfile 5.0 includes Config 3.0, Fault Tolerance 4.0, Rest Client 3.0, Health 4.0, Metrics 4.0, Open Tracing 3.0, Open API 3.0, and JWT propagation 2.0. This beta driver will be used as a compatible implementation for releasing MicroProfile 5.0.
```

Comment:
you might put these features in a list for readability

 MicroProfile 5.0 includes the following features:
 - Config 3.0
 - Fault Tolerance 4.0 etc